src/styles/components/_detailsBanner.scss

```diff
@@ -1,27 +1,20 @@
 .details-banner {
-  line-height: 2.125rem; // 28px
-  letter-spacing: 0.02em; // 2% in Figma
   max-width: 82.875rem; // compensates for 8.5625rem padding on left/right
 
   @media only screen and (min-width: 100rem) {
     margin: auto;
   }
 
-  @media (min-width: $breakpoint-scale-up) {
-    font-size: map.get($font-sizes, "xl");
-    line-height: 1.4; // 70px/16 (get the line-height number from the target-line-height/font-size; 70 / 50)
-  }
-
   h1 {
     margin: 1.875rem 0; // top&bottom: 30px/16; left&right: 0
-    line-height: 2.125rem; // 34px/16
+    line-height: 1.36; // 34px/25
     letter-spacing: 0;
     font-family: Viga, sans-serif;
     font-size: 1.5625rem; // 25px/16
 
     @media (min-width: $breakpoint-scale-up) {
       font-size: map.get($font-sizes, "xl");
-      line-height: 1.4; // 70px/16
+      line-height: 1.34; // 67px/16
```

Comment:
The value's right, but the comment should be 67px/50

Reply:
Hopefully everything is correct now! https://github.com/sparkbox/accessible-components/pull/119/commits/c0efaf021fc223aa3c7ea7e2f5c2b32bcfe55511